openedx/frontend-app-course-authoring
# feat: redirect to unit page if the hit or its parent is a unit

## Description

This PR adds the feature to redirect from a search result to a Unit, in case the hit parent is a Unit, or the hit is a unit itself.

## More Information:

Closes:
- https://github.com/openedx/modular-learning/issues/210

Depends on:
- https://github.com/openedx/edx-platform/pull/34535
- https://github.com/openedx/frontend-app-course-authoring/pull/945

## Testing Instructions

1. Go to the Course Authoring MFE, e.g. http://apps.local.edly.io:2001/course-authoring/home
1. Optionally, browse to a specific course.
1. Click on the search icon in the header
1. Search for a Unit and/or a component inside a unit and click on it
1. You should be redirected to the unit page and the selected component should be highlighted (if any).

---
Private ref: [FAL-3709](https://tasks.opencraft.com/browse/FAL-3709)
